contracts/metadata/metadata.cairo

```diff
@@ -0,0 +1,647 @@
+        const wonder_26 = 112197201601868900173442872257905255795; // The Mythic Trees
+        const wonder_27 = 1882356685885271628290621176910462547826861925; // The Perpetual Ridge
+        const wonder_28 = 1711993432612204213653063014638967; // The Fading Yew
+        const wonder_29 = 112197201602479355747873300554309658995; // The Origin Oasis
+        const wonder_30 = 481883311590596369487344008580053433603419239012; // The Sanctified Fjord
+        const wonder_31 = 438270318760813684155072613920301426; // The Pale Pillar
+        const wonder_32 = 121860869748951841111587458942179520484707200625765; // Sanctum Of The Oracle
+        const wonder_33 = 28722483609443051198127402181533308775525; // The Ethereal Isle
+        const wonder_34 = 438270318759661298254552393728812403; // The Omen Graves
+        const wonder_35 = 438270318760813684155079193911387512; // The Pale Vertex
+        const wonder_36 = 481883311574702770666943468041384486870196644965; // The Glowing Pinnacle
+        const wonder_37 = 1711993432589054777342731153927013; // The Azure Lake
+        const wonder_38 = 481883311566758392789555771262479592161113105011; // The Argent Catacombs
+        const wonder_39 = 28722483609357954936504568885723525114222; // The Dark Mountain
+        const wonder_40 = 6011031307300205428; // Sky Mast
+        const wonder_41 = 1489362693872759468834836248621669; // Infinity Spire
+        const wonder_42 = 28722483609444280708779409888647398517102; // The Exalted Basin
+        const wonder_43 = 1882356685807568525289183711650833565076514163; // The Ancestral Trees
+        const wonder_44 = 1882356685885271628290621176910462496304755300; // The Perpetual Fjord
+        const wonder_45 = 7352955803935814556680353466654377731125102; // The Ancient Lagoon
+        const wonder_46 = 438270318760832371664907227027105138; // The Pearl River
+        const wonder_47 = 31580704707008893635492845615914781084363527406318201; // The Cerulean Reliquary
+        const wonder_48 = 373503018769248307314653188184877340105860803692; // Altar Of Divine Will
+        const wonder_49 = 27352061535142984367278971188825042546277; // Pagoda Of Fortune
+        const wonder_50 = 438270318759684835528781643805323116; // The Oracle Pool
+    }
+}
+
+namespace Uri {
+    func build{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(
+        realm_id: Uint256, realm_data: RealmData, realm_type: felt
+    ) -> (encoded_len: felt, encoded: felt*) {
+        alloc_locals;
+
+        // pre-defined for reusability
+        let left_bracket = Utils.Symbols.LeftBracket;
+        let right_bracket = Utils.Symbols.RightBracket;
+        let inverted_commas = Utils.Symbols.InvertedCommas;
+        let comma = Utils.Symbols.Comma;
+
+        let data_format = 37556871985679581542840396273993309325169359621942828; // data:application/json,
```

Comment:
Hmm, so here I would actually prefer using the single quoted text instead of the numerical encoding.